### PR TITLE
Mitaka merge liberty forward.

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: systest/Makefile [hunk 1]
```diff
@@ -71,47 +72,51 @@ PROJDIR := $(shell dirname $(realpath $(MAKEFILE_DIR)))
 TAGINFO := $(shell git describe --long --tags --first-parent)
 
 # git branch
-BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+BRANCH := liberty
```
Comment (pjbreaux) [inline systest/Makefile]: Why this change?

**Comment by zancas** [comment]
OK @pjbreaux I addressed your comment.

**Comment by zancas** [comment]
@pjbreaux ?
